### PR TITLE
Add KMS quickstart.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: kms/quickstart.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+'use strict';
+
+// [START kms_quickstart]
+// Imports the Google APIs client library
+const google = require('googleapis');
+
+// Your Google Cloud Platform project ID
+const projectId = 'YOUR_PROJECT_ID';
+
+// The "global" zone will list all keys, can be a specific zone if desired
```
Comment (ace-n) [inline kms/quickstart.js]: Grammar nit: The "global" zone will list all keys. **It** can be a specific zone if desired

File: kms/quickstart.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+const zone = 'global';
+
+// Acquires credentials
+google.auth.getApplicationDefault((err, authClient) => {
+  if (err) {
+    console.error('Filed to acquire credentials');
```
Comment (ace-n) [inline kms/quickstart.js]: Typo: F**a**iled

File: kms/system-test/quickstart.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+require(`../../system-test/_setup`);
+
+const proxyquire = require(`proxyquire`).noPreserveCache();
+const google = proxyquire(`googleapis`, {});
+
+function list (callback) {
+  google.auth.getApplicationDefault((err, authClient) => {
+    if (err) {
+      callback(new Error('Filed to acquire credentials'));
```
Comment (ace-n) [inline kms/system-test/quickstart.test.js]: Typo: f**a**iled

**Comment by ace-n** [comment]
LGTM, other than grammatical nits and (some relevant) failing tests.

**Comment by ace-n** [comment]
Failing test output on CircleCI:
```
Uncaught Exception: kms/system-test/quickstart.test.js
  TypeError: google.cloudkms is not a function
    google.auth.getApplicationDefault (kms/system-test/quickstart.test.js:36:29)
    callback (node_modules/googleapis/node_modules/google-auth-library/lib/auth/googleauth.js:42:14)
    node_modules/googleapis/node_modules/google-auth-library/lib/auth/googleauth.js:289:13
```

File: kms/quickstart.js (hunk 1 above)
Comment (jmdobry) [inline kms/quickstart.js]: Fixed

File: kms/quickstart.js (hunk 2 above)
Comment (jmdobry) [inline kms/quickstart.js]: Fixed

File: kms/system-test/quickstart.test.js (hunk 3 above)
Comment (jmdobry) [inline kms/system-test/quickstart.test.js]: Fixed

**Comment by jmdobry** [comment]
Fixed

**Pull request merged**